L1Trigger/L1TTrackMatch/plugins/L1TrackTripletEmulatorProducer.cc

```diff
@@ -277,14 +277,13 @@ void L1TrackTripletEmulatorProducer::produce(Event &iEvent, const EventSetup &iS
 
   if (!event_pass) {
     iEvent.put(std::move(L1TrackTripletContainer), OutputDigisName);
+    iEvent.put(std::move(L1TrackTripletWordContainer), OutputWordName);
     return;
   }
-
   float tripletPx = (pion1 + pion2 + pion3).Pt() * cos((pion1 + pion2 + pion3).Phi());
   float tripletPy = (pion1 + pion2 + pion3).Pt() * sin((pion1 + pion2 + pion3).Phi());
   float tripletPz = (pion1 + pion2 + pion3).Pt() * sinh((pion1 + pion2 + pion3).Eta());
-  float tripletP = (pion1 + pion2 + pion3).Pt() * cosh((pion1 + pion2 + pion3).Eta());
-
+  float tripletP = sqrt( tripletPx*tripletPx + tripletPy*tripletPy + tripletPz*tripletPz + triplet_mass*triplet_mass);
```

Comment:
why is there this change?

Reply:
The line was wrong. The objected wanted the energy not P( I can change also the variable). Wouldnt affect anything in the rate or efficiency because is just the output value, just the mass output was wrong. Now is fixed